### DISCUSSION TITLE
8월 29일 일기

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#42] [team]
다들 퇴근하면서 오늘 감상 남겨줘

**Comment by iamyanghee** [comment]
@mews2000 

"오늘부터 리액트 공부하는 게 맞나 ?"
동적 페이지 정적 페이지 정확한 차이가 뭐지?
자바스크립트 자체를 모르는데 Thymleaf 지금 버리고 리액트 공부하면 속도가 나올까?

**Reply by iamyanghee** [reply]
뭐야 역시 츤츤해
업데이트 해줬어 ㅋㅋ

**Comment by bksooon** [comment]
멘토님을 만났다 마음이 복잡하다...

**Reply by iamyanghee** [reply]
참 잘했다

**Comment by iamyanghee** [comment]
@Lucete28 

"아래의 생각이 들었다"
> ml을 얼마나 할 것인가
> 플레이리스트 생성하는 방식에 대해 기존의 뚝딱에서 검색한 노래와 연관되는 걸 추천해주는 식 괜찮겠다
> 내일 양꼬치 맛있겠다

**Comment by jungssg** [comment]
- 오늘 비가많이와서 기분이 안좋았다
- 음악추천을 어떻게 하면 좋은추천?
- 몰라

**Reply by iamyanghee** [reply]
알잖아요

**Comment by iamyanghee** [comment]
@iamyanghee 


```
내 개발 역량이 딸리는데 어떻게 프로젝트 잘 돌아가게 할 수 있는건지 ... 막막했다
그래서 일단 일기를 도입하기로 했다
근데 얘들이 손꾸락이 고장났는지 대리 타자를 하는데, 꽤나 킹 받는다
뒤에서 손준성이 계속 딴지를 거는데 조만간 맞겠다
여러모로 손이 많이 가는데, 나 손 많이 가는 거 싫은데... ?

이렇게 별 말 저말 이말 다 말랑말랑하게 적었으면들 좋겠는데
다들 또 안 적겠지????

제발 적어줘
````

**Comment by json9101** [comment]
@json9101

비로그인 서비스를 먼저 구상해야하는데 뭐가 있을까?
플레이리스트는 좋은 구상인거같다
기능을 하나씩 같이 구현해 나가는게 좋을꺼같다.
필요 기술 스택들을 좀 빨리 캐치해 나가면서 기간 감안하면서 프로젝트를 진행하는게 좋겠다.
음악 추천할때 머신러닝혹은 딥러닝 외에 다른방법이 있을까?

**Reply by iamyanghee** [reply]
기범님이랑 하는 mvp 끝나면 같이 해요오

**Comment by bksooon** [comment]
```
모든게 짠하고 완성될 수는 없지만 그래도 요행을 바랄때가 있다. 
그게 9월 20일에 발현됐으면 좋겠다
```

**Reply by iamyanghee** [reply]
화이팅화이팅

어제 표정이 안좋은건 그냥 내 손가락이 답답해서 그런거였음
챙겨줘서 고마워 병규 
팀장 버리고 어디 가려구
